torchvision/io/video.py

```diff
@@ -89,8 +95,13 @@ def write_video(filename, video_array, fps: Union[int, float], video_codec="libx
 
 
 def _read_from_stream(
-    container, start_offset, end_offset, pts_unit, stream, stream_name
-):
+    container: "av.container.Container",
+    start_offset: int,
+    end_offset: float,
```

Comment:
Given that `pts_unit` can also be `sec`, meaning that `start_offset` is a float, I think it might be preferable to have `start_offset` being a `float` as well. According to https://stackoverflow.com/questions/50928592/mypy-type-hint-unionfloat-int-is-there-a-number-type
> when an argument is annotated as having type float, an argument of type int is acceptable;

But this is not blocking for this PR, and can be done in a follow-up PR